src/AutoPilotPlugins/PX4/PX4TuningComponentPlaneRate.qml

```diff
@@ -0,0 +1,173 @@
+/****************************************************************************
+ *
+ * (c) 2021 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
+ ****************************************************************************/
+
+import QtQuick          2.3
+import QtQuick.Controls 1.2
+import QtQuick.Layouts  1.2
+
+import QGroundControl               1.0
+import QGroundControl.Controls      1.0
+import QGroundControl.FactSystem    1.0
+import QGroundControl.FactControls  1.0
+import QGroundControl.ScreenTools   1.0
+import QGroundControl.Vehicle       1.0
+
+ColumnLayout {
+    width: availableWidth
+    anchors.fill: parent
+    property alias autotuningEnabled: pidTuning.autotuningEnabled
+
+    GridLayout {
+        columns: 2
+    }
+    PIDTuning {
+        width: availableWidth
+        id:    pidTuning
+
+        property var roll: QtObject {
+            property string name: qsTr("Roll")
+            property var plot: [
+                { name: "Response", value: globals.activeVehicle.rollRate.value },
+                { name: "Setpoint", value: globals.activeVehicle.setpoint.rollRate.value }
+            ]
+            property var params: ListModel {
+                ListElement {
+                    title:          qsTr("Porportional gain (FW_RR_P)")
+                    description:    qsTr("Porportional gain.")
+                    param:          "FW_RR_P"
+                    min:            0.3
+                    max:            1
+                    step:           0.05
+                }
+                ListElement {
+                    title:          qsTr("Differential Gain (FW_RR_D)")
+                    description:    qsTr("Damping: increase to reduce overshoots and oscillations, but not higher than really needed.")
+                    param:          "FW_RR_D"
+                    min:            0.0004
```

Comment:
This needs to be 0, as per default value.
Can you generally go through all ranges and step sizes?

Reply:
Done!